src/test/resources/DataSelectionDevTest.properties

```diff
@@ -1,17 +1,13 @@
 # defined here rather than using the serverUrl property defined in ~/.m2/settings.xml
 # so that this test can be run against a specific ICAT
-icat.url = https://localhost:8181
-
+icat.url=https://localhost:8181
```

Comment:
Here, I'd actually prefer the original file with the spaces around `=` and the blank lines between the different settings.